compose/functions.c

```diff
@@ -1488,6 +1527,43 @@ static int op_compose_group_lingual(struct ComposeSharedData *shared, int op)
   return group_attachments(shared, LINGUAL_TAG, "multilingual");
 }
 
+/**
+ * op_compose_group_related - Group tagged attachments as 'multipart/related' - Implements ::compose_function_t - @ingroup compose_function_api
+ */
+static int op_compose_group_related(struct ComposeSharedData *shared, int op)
+{
+  static const char *RELATED_TAG = "Related parts for \"%s\"";
+  if (shared->adata->menu->tagged < 2)
+  {
+    mutt_error(_("Grouping 'related' requires at least 2 tagged messages"));
+    return IR_ERROR;
+  }
+
+  // ensure Content-ID is set for tagged attachments
+  for (struct Body *b = shared->email->body; b; b = b->next)
+  {
+    if (b->tagged)
```

Comment:
This `if` doesn't have an `else` clause, so we can "return early"

```c
if (!b->tagged)
  continue;
```

This will reduce the indent of the code and mean that the reader doesn't need to hunt for an `else` if the `Body` isn't tagged.